java/app-encryption/src/it/java/com/godaddy/asherah/ConfigReader.java

```diff
@@ -0,0 +1,30 @@
+package com.godaddy.asherah;
+
+public class ConfigReader {
```

Comment:
Rename this class to something like `TestConfiguration` as we really aren't reading anything here